docs/demo/wat2wasm/demo.js

```diff
@@ -52,9 +39,9 @@ var base64El = document.getElementById('base64');
 var binaryBuffer = null;
 var binaryBlobUrl = null;
 
-for (var feature of FEATURES) {
-  var featureEl = document.getElementById(feature);
-  features[feature] = featureEl.checked;
+for (const [f, v] of wabt.allFeatures()) {
```

Comment:
How about `for (const [f, v] of Objet.entries(wasm.FEATURES) } {`?  Then you don't need to new `addFeatures` function at all.